cmd/kubeadm/app/phases/token/bootstrap_test.go

```diff
@@ -120,11 +145,13 @@ func TestCreateBootstrapConfigMapIfNotExists(t *testing.T) {
 			})
 		}
 
-		err = CreateBootstrapConfigMapIfNotExists(client, file.Name())
-		if tc.expectErr && err == nil {
-			t.Errorf("CreateBootstrapConfigMapIfNotExists(%s) wanted error, got nil", tc.name)
-		} else if !tc.expectErr && err != nil {
-			t.Errorf("CreateBootstrapConfigMapIfNotExists(%s) returned unexpected error: %v", tc.name, err)
+		for _, file := range tempFiles {
+			err := CreateBootstrapConfigMapIfNotExists(client, file.Name())
```

Comment:
Just curious, could the code be refactored to accept a stream, instead of a file name? If so, then you could pass in a string as a stream, instead of creating temp files.